dspy/clients/embedding.py

```diff
@@ -1,8 +1,8 @@
 import litellm
 import numpy as np
+from typing import Callable, List, Union, Optional
 
-
-class Embedding:
+class Embedder:
```

Comment:
I chose name `Embedding` because that seems to be the name for most packages:

- Langchain uses `XXXEmbeddings`
- LlamaIndex uses `XXXEmbedding`
- OpenAI python SDK uses `client.embeddings.create`

I didn't do a thorough search, Is there some package that is using the name `Embedder`?